__tests__/pet.test.js

```diff
@@ -0,0 +1,187 @@
+const Pet = require('../src/pet');
+const Parent = require('../src/pet');
+
+describe('constructor',()=>{
+    it('returns an object',()=>{
+        expect(new Pet('Fido')).toBeInstanceOf(Object);
+    })
+   
+
+    it('sets the name property',()=>{
+        const myPet = new Pet('Fido');
+        expect(myPet.name).toEqual('Fido');
+    })
+
+
+    it('has an initial age of 0',()=>{
+        const myPet = new Pet('Fido');
+        expect(myPet.age).toEqual(0);
+    })
+})
+
+describe('growUp',()=>{
+    it('increments the age by 1',()=>{
+        const myPet = new Pet('Fido');
+        myPet.growUp();
+        expect(myPet.age).toEqual(1);
+        expect(myPet.hunger).toEqual(5);
+        expect(myPet.fitness).toEqual(7);
+    })
+})
+
+
+
+describe('fitness',()=>{
+  it('increases fitness by walking by 4',()=>{
+    const myPet=new Pet('Fido');
+    myPet.fitness=4;
+    myPet.walk();
+    expect(myPet.fitness).toEqual(8);
+  })
+
+})
```

Comment:
I would add more tests to check that fitness cannot go above `MAX_FITNESS`, i.e.
```
it('cannot go above a fitness of 10', () => {
  ...
});
```

Reply:
should also check that it throws and error if the pet is dead, as you shouldn't be able to walk a dead pet